presto-main/src/main/java/com/facebook/presto/sql/planner/plan/AssignUniqueId.java

```diff
@@ -72,4 +74,11 @@ public Symbol getIdColumn()
     {
         return visitor.visitAssignUniqueId(this, context);
     }
+
+    @Override
+    public PlanNode replaceChildren(List<PlanNode> newChildren)
```

Comment:
"sources" is a legacy term. It used to be accurate a *long* time ago, before we even had support for "index joins". Then, with that feature, and now with Apply, it can mean either source or subplan. So "child" is more correct now.